deployment/provision_vagrant_vm.sh

```diff
@@ -36,12 +36,22 @@ service redis-server restart
 # install apache
 apt-get -q install -y apache2 apache2-dev libapache2-mod-wsgi-py3
 
-# With docker for mac, root will own the mount point (uid=0). chmod does not touch the host file system in these cases.
-OWNER=$(stat -c %U "$MOUNTPOINT/evap")
-if [ "$OWNER" == "root" ]; then chown -R 1042 "$MOUNTPOINT"; fi
+cp /etc/skel/.bashrc /home/$USER/
+# link the mounted evap folder from the home directory:
+OWNER=$(stat -c %u "$MOUNTPOINT/evap")
+if [ "$OWNER" != 1042 ]; then
+  #  if, for any reason, vagrant failed to mount evap under the correct uid, use bindfs to link it to /opt/evap using the wanted user id
+  #  If you are using docker: Make sure /dev/fuse of your host is forwarded into the container
+  apt-get -q install -y bindfs
+  mkdir -p "$REPO_FOLDER"
+  bindfs --map="$OWNER/1042:@$OWNER/@1042" "$MOUNTPOINT" "$REPO_FOLDER" || exit 1
+  echo "sudo bindfs --map=$OWNER/1042:@$OWNER/@1042 '$MOUNTPOINT' '$REPO_FOLDER'" >> /home/$USER/.bashrc
+else
+  ln -s "$MOUNTPOINT" "$REPO_FOLDER"
+fi
```

Comment:
Should we even keep this? It might make debugging more complicated if we have two possible setups. Or does bindfs break if we map 1042:1042 or so?

Reply:
bindfs certainly supports this. My idea was to keep the 'old' (simpler?) behavior for 'standard' environments

Reply:
I would prefer if we would be confident that the bindfs solution works in all cases and only have the one setup, but if you prefer having working setups definitely keep working, that is also fair :+1:

Reply:
Does the ubuntu virtualbox base image already contain bindfs (-> is apt install a noop)? If yes, I'd agree that always using bindfs simplifies stuff without adding bloat/requirements and I'd vote in favor of just always using it.

(If it has to be installed first, I'm neutral.)

Reply:
`bindfs` is not installed by default, so I will switch to bindfs. I'm only a bit concerned with fuse and other "exotic" setups, like wsl on windows (even though wsl 2 should support fuse (https://unix.stackexchange.com/a/639820))